tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestUnionQuery.java

```diff
@@ -43,658 +49,760 @@ public TestUnionQuery() {
     super(TajoConstants.DEFAULT_DATABASE_NAME);
   }
 
-  @Test
   /**
    * S (SA U SA) O
    */
+  @Test
+  @SimpleTest
   public final void testUnionAll1() throws Exception {
-    ResultSet res = executeQuery();
-    assertResultSet(res);
-    cleanupQuery(res);
+    Optional<TajoResultSetBase[]> existing = runSimpleTests();
```

Comment:
How about making the below testing code as a separate method to reuse in every test of this class?